File: 3-Chapter-4.1/Revision_Material/nine.py
def compare_length(string: str) -> None:
    if len(string) > 10:
        print(string + "...and I love ice cream")
    elif len(string) < 10:
        print(string + "...and I love chocolate brownies!")
    else:
        print("Goodbye")


user_string = input("Enter a phrase: ")
compare_length(user_string)

File: 3-Chapter-4.1/Revision_Material/nine.cs
using System;
namespace CSharpWork
{
    public class Nine
    {
        public Nine()
        {
            string userString = Prompt();
            CompareLength(userString);
        }

        static void CompareLength(string str)
        {
            if (str.Length > 10)
            {
                Console.WriteLine(str + "...and I love ice cream");
            }
            else if (str.Length < 10)
            {
                Console.WriteLine(str + "...and I love chocolate brownies!");
            }
            else
            {
                Console.WriteLine("Goodbye");
            }
        }

        static string Prompt()
        {
            Console.WriteLine("Enter a phrase:");
            return Console.ReadLine();
        }
    }
}
